### ISSUE TITLE
[1.16.5] [Incompatibility] Dungeons Mobs breaks Special Mobs attributes config.

### ISSUE THREAD

**Issue by Sazuzaki** [Thelnfamous1/Dungeons-Mobs#185]
Heyo, just reporting this mod breaks the mob attributes config for Special Mobs. I am unable to edit attributes such as max health or damage through the normal config files with this mod installed.

Not sure if you need any log files, but lmk if you do.

**Comment by Thelnfamous1** [comment]
Yes, do post logs. However, I am currently focused on bugfixing 1.19.2, intending to get it into a stable place before bringing those fixes downstream and also addressing any version-specific issues. I cannot promise I will get to fixing this specific issue anytime soon.
